### ISSUE TITLE
Pokémon BDSP - New Icons

### ISSUE THREAD

**Issue by WordeurFTB** [msikma/pokesprite#111]
There are new icons with the new games releasing later this month.
Thanks to dataminers we got some visuals.

<details>
  <summary>Spoiler warning</summary>

Source : https://twitter.com/mattyoukhana_/status/145793379637279949
![FDuerkqX0AQ9na5](https://user-images.githubusercontent.com/23153824/140964287-027c9cfa-f384-41a7-9f6e-02d829f531a2.jpg)

</details>

Do you think, msikma, that you will cover these in your project ?

**Comment by msikma** [comment]
Yeah, I'll gladly add these to the project, although I will need a lossless quality version with transparent background. I'll see if I can get that. (Or if anyone happens to have this, feel free to share—preferably cut into separate files rather than a single sheet.)

Given that this is a completely new set, rather than minor updates, this will be a new set of sprites entirely in a separate directory.

Would you happen to know if these are already datamined from one of the advance/review copies of the retail release, or is this from like an early leak or demo? Since I'm wondering if maybe the retail release will have more sprites or updated versions.

**Comment by msikma** [comment]
I've got a copy of all the sprites from BDSP now (box sprites plus items and other things) thanks to @realph1l on Twitter, so I'll start working on this soon and have some updates.

**Comment by WordeurFTB** [comment]
Ok, I've juste seen that the Twitter link is now dead somehow so here is the link to the image it was redirecting to :
https://cdn.discordapp.com/attachments/677045461709881344/907492226966368256/gems_sprites_montage.png
And as for the items all I could find was this poor quality sheet :
https://pbs.twimg.com/media/FDufPZtWEAEfsEd?format=png&name=900x900
If you found resources with a better quality it's nice !

**Comment by sora10pls** [comment]
Hey, I remember I was mentioned in a tweet regarding this issue and would be happy to share texture assets with you, @msikma.

What would be the best place to contact you? Are Twitter DMs fine like last time, or would you prefer somewhere else?

**Comment by AlamosIT** [comment]
If it could be useful, here there are all the clean rips of textures of BDSP v1.1.1

https://mega.nz/file/QcV2UDZL#97R8ede65X_L6TQS-NBd6X4-S9YIyujxkype2dkBsA4

[Source](https://twitter.com/mattyoukhana_/status/1462809268739088390)

**Comment by guitaristtom** [comment]
Looks like the Twinkling Star Ribbon is missing from the repo still.

**Comment by JosephTico** [comment]
Ok, we now should seriously consider merging this, but in a different way, as we now know that Gen 9 will also use the same BDSP icons.

So maybe instead of adding them as BDSP, they should be added as gen 9 icons.

**Comment by msikma** [comment]
Yeah, the fact that all new BDSP sprites have different sizes is kind of annoying because that would mean Gen 8 images can have either a predetermined size, or a totally random size that you can't know ahead of time. Both require a specific approach when using it. I don't know if the BDSP sprites are all at least rescaled to specific sizes on screen, though (made to fit in a square?).

So maybe these should be called "Gen 8b" or "Gen 9" since as @JosephTico mentioned they will be reused for Gen 9.
